apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -167,33 +166,28 @@ class PageTemplateModal extends Component {
 
 	static getDefaultSelectedTemplate = ( props ) => {
 		const blankTemplate = get( props.templates, [ 0, 'slug' ] );
-		let previouslyChosenTemplate = props._starter_page_template;
+		const previouslyChosenTemplate = props._starter_page_template;
 
-		// Usally the "new page" case
+		// Usually the "new page" case
 		if ( ! props.isFrontPage && ! previouslyChosenTemplate ) {
 			return blankTemplate;
 		}
 
-		// Normalize "home" slug into the current theme.
-		if ( previouslyChosenTemplate === 'home' ) {
-			previouslyChosenTemplate = props.theme;
-		}
-
-		const slug = previouslyChosenTemplate || props.theme;
-
-		if ( find( props.templates, { slug } ) ) {
-			return slug;
-		} else if ( find( props.templates, { slug: DEFAULT_HOMEPAGE_TEMPLATE } ) ) {
-			return DEFAULT_HOMEPAGE_TEMPLATE;
-		}
-		return blankTemplate;
+		// if the page isn't new, select "Current" as the default template
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **100** times:
`translate( 'Current' )` *ES Score: 10*